javascript/javascript1/week3/week3-guzide/warmups/script.js

```diff
@@ -0,0 +1,84 @@
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+let removeElement = names.indexOf(nameToRemove);
+names.splice(removeElement, 1);
+// Code done
+
+console.log(names);
+console.log("-----------------------------------------");
+
+//When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+function travelDuration(travelinfo) {
+  const rawTravelDuration = travelinfo.destinationDistance / travelinfo.speed;
```

Comment:
This is a bit verbose but gets the job done! ⭐ If you have the time, I'd suggest looking into getting this down to a fewer lines. 😄 Feel free to reach out if you get stuck!